lib/workers/branch/index.js

```diff
@@ -175,6 +175,37 @@ async function processBranch(branchConfig, prHourlyLimitReached, packageFiles) {
       );
     }
 
+    let isStableVersionReleaseTS = true;
+    if (config.stabilityThreshold) {
+      const parsedStabilityThreshold = Duration.fromISO(
+        config.stabilityThreshold
+      );
+      if (parsedStabilityThreshold.invalid) {
+        throw new Error(
+          `Stability threshold error: ${parsedStabilityThreshold.invalid.explanation}`
+        );
+      }
+
+      const parsedLatestUpgradeTimestamp = DateTime.fromISO(
+        config.latestUpgradeTimestamp
```

Comment:
Please check what happens if there are *no* timestamps present.  npm normally has them but many others do not. When there are no timestamps, we should default to not limiting/thresholding and let every PR through.

Reply:
added a check for `config.releaseTimestamp` too where i check for `config.stabilityThreshold` so there's no issues there now. if one of them is not set, the PR will go through